src/components/ProposalListTablePanel.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { Bars3BottomLeftIcon } from '@heroicons/react/24/solid';
+import {
+  Panel,
+  PanelHeader,
+  PanelBody,
+  PanelActions,
+} from './Panel';
+import { ProposalListTable } from './ProposalListTable';
+import { Button } from './Button';
+
+interface ProposalListTablePanelProps {
+  fieldNames: Record<string, string>;
+  proposals: {
+    id: string;
+    values: Record<string, string[]>;
+  }[];
+}
+
+// For now, we are hard-coding this list.
+// In the future, this will be user-configurable.
+const DEFAULT_COLUMNS = [
+  'organization_name',
+  'organization_tax_id',
+  'organization_city',
+  'organization_state_province',
+  'organization_country',
+  'organization_website',
+  // 'organization_mission_statement', // FIXME: Disabled until we have wrap
+  'organization_start_date',
+  'organization_operating_budget',
+  'proposal_title', // May need to conditionally use `proposal_name`?
+  'proposal_summary',
+  'proposal_amount_requested',
+  'proposal_budget',
+  'proposal_fiscal_sponsor_name',
+  'proposal_start_date',
+  'proposal_end_date',
+  'proposal_location_of_work',
+];
+
+export const ProposalListTablePanel = ({
+  fieldNames,
+  proposals,
+}: ProposalListTablePanelProps) => {
+  const [wrap, setWrap] = useState(false);
+
+  const handleWrapClick = () => setWrap((previous) => !previous);
+
+  return (
+    <Panel>
+      <PanelHeader>
+        <PanelActions>
+          <input
+            type="text"
+            placeholder="Search…"
+            className="input"
+          />
```

Comment:
I know that you were instructed to do this, but it feels Not Great - there's no indicator as to why it won't work, and especially if we're going to be handing out logins soon, I feel like this is likely to be distracting. Can we give it a popup or `disable` it or give it a `disabled` search button or something?

Reply:
Agree all around. I think Karl's actual suggestion was that, upon submitting, we display a "coming soon" sort of message. I just punted on that complexity.

What about a dreaded `alert()` on submit?

I'm hoping to avoid a submit button until I style one I like (I have Ideas™); do you think just enter-to-submit is sufficient UI, or do we need a stand-in submit button?

Reply:
Can you please add that button, then, and squash it into commit 2c425aa? I think that's the last thing this needs!

Reply:
On it!